src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -149,6 +149,7 @@ export interface SavedObjectsServiceSetup {
  */
 export interface InternalSavedObjectsServiceSetup extends SavedObjectsServiceSetup {
   status$: Observable<ServiceStatus<SavedObjectStatusMeta>>;
+  getTypeRegistry: () => ISavedObjectTypeRegistry;
```

Comment:
Not strictly necessary, but makes the test faster since I don't have to wait for `start` to complete

Reply:
Yep, I'll just add that to the docstring to make it clear